src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball{
     public String getRandomColor() {
```

Comment:
The method getRandomColor should have an access modifier. In this case, it seems like it should be public.